app/experimenter/experiments/models.py

```diff
@@ -545,7 +545,7 @@ class ExperimentVariant(models.Model):
     is_control = models.BooleanField(default=False)
     description = models.TextField(default="")
     ratio = models.PositiveIntegerField(default=1)
-    value = JSONField(blank=True, null=True)
+    value = models.CharField(max_length=10240, blank=False, null=True)
```

Comment:
@tiftran Ah my bad I should've said TextField here, can you switch it?

Reply:
Would the max_length be needed it's a TextField?

Reply:
Oh sorry you asked here, I didn't see, no for TextField the only thing max_length does is set the size of the input field, so you usually just omit it and specify it on the actual widget (which you've done correctly already)